src/NServiceBus.Core.Tests/API/APIApprovals.cs

```diff
@@ -16,7 +16,7 @@ public class APIApprovals
         [MethodImpl(MethodImplOptions.NoInlining)]
         public void ApproveNServiceBus()
         {
-            var assemblyPath = Path.GetFullPath(typeof(IBusSessionFactory).Assembly.Location);
+            var assemblyPath = Path.GetFullPath(typeof(IBusSession).Assembly.Location);
```

Comment:
That was randomly used by me. It can be any

On Monday, January 25, 2016, Tim Bussmann <notifications@github.com> wrote:

> In src/NServiceBus.Core.Tests/API/APIApprovals.cs
> <https://github.com/Particular/NServiceBus/pull/3356#discussion_r50701615>
> :
>
> > @@ -16,7 +16,7 @@ public class APIApprovals
> >          [MethodImpl(MethodImplOptions.NoInlining)]
> >          public void ApproveNServiceBus()
> >          {
> > -            var assemblyPath = Path.GetFullPath(typeof(IBusSessionFactory).Assembly.Location);
> > +            var assemblyPath = Path.GetFullPath(typeof(IBusSession).Assembly.Location);
>
> I don't now why exactly IBusSessionFactory ended up as the "NSB.Core
> marker interface" but one of the endpoint interfaces/classes would seem
> more appropriate to me, e.g.IEndpointInstance or Endpoint?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3356/files#r50701615>.
>